src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/AvoidSingleUseOfLocalJsonSerializerOptions.cs

```diff
@@ -103,6 +103,12 @@ private static bool IsLocalUsedAsArgumentForJsonSerializerOnly(IObjectCreationOp
                     continue;
                 }
 
+                // Symbol is declared in a parent scope, so this implies that options are re-used.
```

Comment:
For my education, would this also disable the diagnostic in something like this?
```C#
var options = new JsonSerializerOptions();
{
     JsonSerializer.Serialize(value, options);
}
```
If that's the case FWIW I don't think that's a problem we would need to address.